### PR TITLE
#137005181 User on-boarding

### PR DESCRIPTION
**Pull request by ghost**
#### What does this PR do?

This PR adds a brief introductory tour to reaction commerce

#### Description of Task to be completed?

Users are given a brief tour about the application either as an admin or a guest user.

#### How should this be manually tested?

The take tour link is available on the user account drop down on the homepage for authenticated users, and as a button on the nav bar for unauthenticated users.

#### What are the relevant pivotal tracker stories?

137005181 - Users should be quickly and effectively onboarded onto the app

### CODE REVIEW

**Comment by raphaelakpan** [comment]
Nice work!

However, you did not include Screenshot(s) for our perusing. Is this not a feature that we can see?

File: imports/plugins/core/layout/client/templates/layout/header/header.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
       {{> CoreNavigationBar coreNavProps}}
     </div>
 
+
```
Comment (oagboola) [inline imports/plugins/core/layout/client/templates/layout/header/header.html]: why is there need for this empty line?

File: imports/plugins/core/ui-navbar/client/components/navbar/navbar.html [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   <div class="rui navbar">
     <div class="showmenu">{{> button icon="bars" onClick=onMenuButtonClick}}</div>
     {{> coreNavigationBrand}}
     <div class="menu">
       {{> tagNav tagNavProps}}
     </div>
-    
+
+    {{#unless currentUser}}
+    <div id="takeTour" class="tour">{{> React tourButton}}</div>
```
Comment (oagboola) [inline imports/plugins/core/ui-navbar/client/components/navbar/navbar.html]: fix indentation

File: settings/dev.settings.json [hunk 3]
```diff
@@ -2,9 +2,9 @@
   "ROOT_URL": "",
   "MAIL_URL": "",
   "reaction": {
-    "REACTION_USER": "",
-    "REACTION_AUTH": "",
-    "REACTION_EMAIL": ""
+    "REACTION_USER": "Admin",
+    "REACTION_AUTH": "uyi",
+    "REACTION_EMAIL": "uyi@uyi.com"
```
Comment (oagboola) [inline settings/dev.settings.json]: are these details supposed to be public?

**Comment by oagboola** [comment]
You may need to fix houndCi settings from flagging ES6 syntax

**Pull request merged**